devicetypes/sinope-technologies/dm2550zb-sinope-dimmer.src/dm2550zb-sinope-dimmer.groovy

```diff
@@ -0,0 +1,336 @@
+/**
+Copyright Sinopé Technologies
+1.3.0
+SVN-571
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+**/
+
+preferences {
+	input("MinimalIntensityParam", "number", title:"Light bulb minimal intensity (1..10) (default: blank)", range:"1..10", description:"optional")
+    // when the is at a low value, some bulbs may flicker for some technical reasons. to prevent that behaviour. writting this parameter will increase the minimal value
+    // of the dimmer's become a little bit higher so the load doesn't start flickering when the level is low.
+
+    input("LedIntensityParam", "number", title:"Indicator light intensity (1..100) (default: blank)", range:"1..100", description:"optional")
+	input("trace", "bool", title: "Trace", description: "Set it to true to enable tracing")
+	// input("logFilter", "number", title: "Trace level", range: "1..5",
+	// 	description: "1= ERROR only, 2= <1+WARNING>, 3= <2+INFO>, 4= <3+DEBUG>, 5= <4+TRACE>")
+}
+
+metadata {
+    definition (name: "DM2550ZB Sinope Dimmer", namespace: "Sinope Technologies", author: "Sinope Technologies",  ocfDeviceType: "oic.d.switch")
```

Comment:
Could You try integrating Your device here ?
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy
(by adding a fingerprint only and configuring those parameters with default values)

Reply:
@aguilbault-Sinope Let's continue in another pull request. Please, make sure to mention the new PR here.

Reply:
@KKlimczukS new pull-request created here : https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/77841

Sorry for the inconvenience and thanks in advance!

Reply:
@aguilbault-Sinope will You also issue a new Request in Developer Workspace ? (I noticed the current one was cancelled)
cc: @k-morawski

Reply:
@KKlimczukS Yeah it's already done. I had to cancel the current one  to be able to create a new request from the Developer Workspace.